includes/core/CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ There are a few possible paths when it comes to services:
 
 1. __Create a facade for an existing service:__ Create a new service class within `core/service`, which simply facades the [existing service](service/customer-service.md). Doing so will allow us to modify the facade in the future, keeping existing methods with the same parameters as existing ones.
 This is what was done with the [customer service](service/customer-service.md), and is the recommended way if a certain feature requires access to an existing service quickly.
-2. __Move an existing service to the core directory:__ This should be done with consideration how the service could change in the future, and whether it is core to the gateway. If it more suitable to an extension (ex. [Multi-Currency](https://woocommerce.com/document/woocommerce-payments/currencies/multi-currency-setup/?quid=92bb9bc4a89c89c9445c87865165e025)), or a consumer (ex. [WooPay](https://woocommerce.com/documentation/woopay/)), it likely needs to be somewhere else.
+2. __Move an existing service to the core directory:__ This should be done with consideration how the service could change in the future, and whether it is core to the gateway. If it more suitable to an extension (ex. [Multi-Currency](https://woocommerce.com/document/woopayments/currencies/multi-currency-setup/)), or a consumer (ex. [WooPay](https://woocommerce.com/document/woopay-customer-documentation/)), it likely needs to be somewhere else.
```

Comment:
For WooPay, I think we would use this link https://woocommerce.com/documentation/products/woopay/ 

`Consumer` here is used in the technical context that "WooPay" extends "WooPayments".  

Also note that, this is a tech doc for devs. 

```suggestion
2. __Move an existing service to the core directory:__ This should be done with consideration how the service could change in the future, and whether it is core to the gateway. If it more suitable to an extension (ex. [Multi-Currency](https://woocommerce.com/document/woopayments/currencies/multi-currency-setup/)), or a consumer (ex. [WooPay](https://woocommerce.com/documentation/products/woopay/)), it likely needs to be somewhere else.
```

Reply:
Ah, good catch! Will commit that.